### PR TITLE
CLDR-13980 v46 ddl: Seed data for Kazakh (Arabic)

### PR DESCRIPTION
**Pull request by MuratKaribai**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13980
- [x] Updated PR title and link in previous line to include Issue number

New locale request for Kazakh (Arabic)

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by MuratKaribai** [comment]
@srl295 please look at the request

File: seed/main/kk_Arab.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		<ellipsis type="word-final">{0} …</ellipsis>
+		<ellipsis type="word-initial">… {0}</ellipsis>
+		<ellipsis type="word-medial">{0} … {1}</ellipsis>
+		<moreInformation>?</moreInformation>
+		<parseLenients scope="date" level="lenient">
+			<parseLenient sample="-" draft="contributed">↑↑↑</parseLenient>
+			<parseLenient sample=":" draft="contributed">↑↑↑</parseLenient>
+		</parseLenients>
+	</characters>
+		<minimalPairs>
```
Comment (srl295) [inline seed/main/kk_Arab.xml:43]:
```suggestion
        <numbers>
		<minimalPairs>
```

File: seed/main/kk_Arab.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			<pluralMinimalPairs count="one">سەبەتتە {0} الما بار. ول سىزدىكى مە؟</pluralMinimalPairs>
+			<pluralMinimalPairs count="other">سەبەتتە {0} الما بار. ولار سىزدىكى مە؟</pluralMinimalPairs>
+			<ordinalMinimalPairs ordinal="many">{0}- بۇرىلىستان وڭعا بۇرىلىڭىز.</ordinalMinimalPairs>
+			<ordinalMinimalPairs ordinal="other">{0}- بۇرىلىستان وڭعا بۇرىلىڭىز.</ordinalMinimalPairs>
+		</minimalPairs>
```
Comment (srl295) [inline seed/main/kk_Arab.xml:48]:
```suggestion
		</minimalPairs>
       </numbers>
```

**Comment by MuratKaribai** [comment]
@srl295 For kk-Latn and changing kk in kk-Cyrl, do I need to create a new branch, or can I add in this branch?

**Comment by srl295** [comment]
> @srl295 For kk-Latn and changing kk in kk-Cyrl, do I need to create a new branch, or can I add in this branch?

There's no need to create a new branch.  Actually, fix the suggestion I made above, and the rest will probably need to be discussed in committee

**Comment by MuratKaribai** [comment]
> There's no need to create a new branch. Actually, fix the suggestion I made above, and the rest will probably need to be discussed in committee

Please check

**Comment by MuratKaribay** [comment]
@srl295 now I need to merge all the commits into a single commit?

**Comment by MuratKaribay** [comment]
@srl295 can I add keyboard data, annotation, and romanization to this branch?
And you can find out when to add the kk_arab locale and how long it will take?

@macchiati can I get permission to edit kk_arab locale data in the Survey Tool?
I have a guest account.

**Comment by MuratKaribay** [comment]
@srl295 please can you answer the questions listed above?

**Comment by MuratKaribay** [comment]
> -  minimalPairs go under <numbers> (submitted a requested change)
> - there's a `kk_Arab` in the exemplars/main directory. Please compare to that as well and delete it from your branch.
> - kk needs to be made multi script, so `kk` will need to change to be `kk-Cyrl` . This is more complex, and is why there are default content issues.
> - This also relates to the upcoming `kk-Latn` https://unicode-org.atlassian.net/browse/CLDR-10736 


@srl295 please can you check?

**Comment by srl295** [comment]
@MuratKaribay I think this should be post 38. I think we need to look closely at the kk-Latn effect here - as we add kk-Arab, I think we should see if we can also add kk-Latn at the same time.

**Comment by srl295** [comment]
@MuratKaribay can you rebase?

**Comment by srl295** [comment]
> can I add keyboard data, annotation, and romanization to this branch?

make it separate.

**Comment by srl295** [comment]
@MuratKaribay  PR still needs rebase. Will move this to v42

**Comment by srl295** [comment]
@MuratKaribai  ping?  can you rebase this?

**Comment by srl295** [comment]
@MuratKaribai this will need to move to v43 if it isn't rebased in the next few days

**Comment by srl295** [comment]
@MuratKaribai Closing this. Note that 'seed' is going away, so when you reopen this, just update common/main not seed/main.

**Comment by srl295** [comment]
@MuratKaribai  can you rebase this into common/main?

**Review by srl295 — APPROVED** [review]

**Review by srl295 — DISMISSED** [review]

**Comment by DavidLRowe** [comment]
Resolving a conflict in supplementalMetadata.xml resulted in a commit message which lacks the ticket number. This can be fixed by squashing the commits, but better to wait for other tests to pass first.

**Comment by DavidLRowe** [comment]
@srl295 Seems to pass tests now, after I removed the empty auxiliary exemplar and resolved a conflict in supplementalMetadata.xml.

**Comment by srl295** [comment]
Thanks… Looks like the merge commit had the wrong message though.

 @MuratKaribai 
 can you squash again?

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
> Resolving a conflict in supplementalMetadata.xml resulted in a commit message which lacks the ticket number. This can be fixed by squashing the commits, but better to wait for other tests to pass first.

https://github.com/unicode-org/cldr/commit/a1173e9cabe891921f288b271d8779e8fb966f70 you can fix the commit message when doing the merge, but you only get one chance because we maintainers can only add commits not modify them…

**Comment by MuratKaribai** [comment]
@srl295 could you please help me with squash, as I opened this PR 4 years ago and have forgotten a lot of things. Maybe I need to enter the data through Survey Tool so that I don't have any problems with PR

**Comment by srl295** [comment]
> @srl295 could you please help me with squash, as I opened this PR 4 years ago and have forgotten a lot of things. Maybe I need to enter the data through Survey Tool so that I don't have any problems with PR

We MUST get this PR in before we can enter data in the survey tool. 

Please just go to https://us-central1-dev-infra-273822.cloudfunctions.net/unicode-github-bot/info/unicode-org/cldr/589 and check the box to authorize squashing the branch. 

Then we can merge

**Review by DavidLRowe — APPROVED** [review]

**Comment by srl295** [comment]
@MuratKaribai thank you for your patience ! Reply on the thread about the account, you should be able to get a kk-Arab account now.

**Pull request merged**